connect/config.py

```diff
@@ -64,21 +64,18 @@ class Settings(BaseSettings):
     # Connect
     connect_cert: str = application_cert_path + '/lfh.pem'
     connect_cert_key: str = application_cert_path + '/lfh.key'
-    lfh_id: str = host_name
-
-    # logging
-    logging_config_path: str = 'logging.yaml'
+    connect_lfh_id: str = host_name
+    connect_logging_config_path: str = 'logging.yaml'
+    # external FHIR server URL or None
```

Comment:
Looks a bit weird to me to not have the blank line above the external fhir server config, but it is all part of the connect config...And I refer to those lines quite a bit, so glad to see they are still there.